models/users.js

```diff
@@ -107,6 +133,75 @@ Meteor.methods({
   },
 });
 
+if (Meteor.isServer) {
+  Meteor.methods({
+    inviteUserToBoard(username, boardId) {
+      check(username, String);
+      check(boardId, String);
+
+      const inviter = Meteor.user();
+      const board = Boards.findOne(boardId);
+      const allowInvite = inviter &&
+          board &&
+          board.members &&
+          _.contains(_.pluck(board.members, 'userId'), inviter._id) &&
+          _.where(board.members, {userId: inviter._id})[0].isActive &&
+          _.where(board.members, {userId: inviter._id})[0].isAdmin;
+      if (!allowInvite) throw new Meteor.Error('error-board-notAdmin');
+
+      this.unblock();
+
+      const posAt = username.indexOf('@');
+      let user = Users.findOne((posAt >= 0) ? {emails: {$elemMatch: {address: username}}} : { username });
+      if (user) {
+        if (user._id === inviter._id) throw new Meteor.Error('error-user-notAllowSelf');
+      } else {
+        if (posAt <= 0) throw new Meteor.Error('error-user-doesNotExist');
+
+        const email = username;
+        username = email.substring(0, posAt);
+        const newUserId = Accounts.createUser({ username, email });
+        if (!newUserId) throw new Meteor.Error('error-user-notCreated');
+        // assume new user speak same language with inviter
+        if (inviter.profile && inviter.profile.language) {
+          Users.update(newUserId, {
+            $set: {
+              'profile.language': inviter.profile.language,
+            },
+          });
+        }
+        Accounts.sendEnrollmentEmail(newUserId);
+        user = Users.findOne(newUserId);
+      }
+
+      board.addMember(user._id);
+      user.addInvite(boardId);
+
+      try {
+        const rootUrl = process.env.ROOT_URL || '';
```

Comment:
I think it better to read it from `Meteor.absoluteUrl.defaultOptions.rootUrl` than from the `process.env`